src/components/tabs/tabs.js

```diff
@@ -168,10 +170,12 @@ class Tab extends ContentSwitcher {
       selectorTrigger: `.${prefix}--tabs-trigger`,
       selectorTriggerText: `.${prefix}--tabs-trigger-text`,
       selectorButton: `.${prefix}--tabs__nav-item`,
+      selectorButtonEnabled: `.${prefix}--tabs__nav-item:not([disabled])`,
       selectorButtonSelected: `.${prefix}--tabs__nav-item--selected`,
       selectorLink: `.${prefix}--tabs__nav-link`,
       classActive: `${prefix}--tabs__nav-item--selected`,
       classHidden: `${prefix}--tabs__nav--hidden`,
+      classOpen: `${prefix}--tabs-trigger--open`,
```

Comment:
Seems to reflect this to the test, as we did for `selectorButtonEnabled`.

Reply:
@asudoh you were right. I added testing for the new class and it passed now. Thank you! 😄